### ISSUE TITLE
Cannot fully remove Tailscale to fix service issue

### ISSUE THREAD

**Issue by traisjames** [tailscale/tailscale#11712]
### What is the issue?

When trying to connect to Tailscale tonight I got the message in the taskbar "Tailscale: Windows service not running" even though I can see that it is.  I tried to follow the post from https://github.com/tailscale/tailscale/issues/2282, including uninstalling Tailscale before reinstalling.  After reinstalling my account was still showing on my computer and still showing the error.  That is when I made the bug report 1st: BUG-94233e6a8a757fa144271a858e748b10a8564006937245cae79267fcda81a30e-20240413034441Z-f761ceaac83115e3.  I attempted to search for and remove every tailscale entry in regedit.  Eventually I got into the Tailscale CLI docs, used Tailscale logout.  Now I get cert errors when signing in via CLI, nothing when using the GUI.  That results in this second bug report:
BUG-94233e6a8a757fa144271a858e748b10a8564006937245cae79267fcda81a30e-20240413044928Z-b67f487aa4ab5c03
Either way, in my attempt to fully purge to reset tailscale, I cannot get it fully removed.
 
System:
Edition	Windows 11 Home
Version	21H2
Installed on	‎8/‎3/‎2022
OS build	22000.2538
Experience	Windows Feature Experience Pack 1000.22001.1000.0

Tailscale 1.64.0

### Are there any recent changes that introduced the issue?

Most recent Windows update 2023-10 Cumulative Update for .NET Framework 3.5, 4.8 and 4.8.1 for Windows 11 for x64 (KB5031225) on 3/7/2024, Tailscale used successfully more recently than that.

### OS

Windows

### OS version

Windows 11 Home 21H2

### Tailscale version

1.64.0

### Other software

iCloud for Windows, but removed

### Bug report

1st: BUG-94233e6a8a757fa144271a858e748b10a8564006937245cae79267fcda81a30e-20240413034441Z-f761ceaac83115e3, 2nd: BUG-94233e6a8a757fa144271a858e748b10a8564006937245cae79267fcda81a30e-20240413044928Z-b67f487aa4ab5c03

**Comment by scropion86** [comment]
i have the same issue , and posted a bug report `BUG-d3de65f0c91f7756cc339786e90336e829d9cbd04156fefd1906b8d8ebaf87ec-20240424053550Z-b67bfd8b2ccd5e62`
and after fiddling with the settings and deleting manually all registry with tailscale on it.

i have found that my android is also not able to connect so I disabled the wifi and used mobile data and was able to connect.
then disconnected my laptop wif and ethernet and connect it to my mobile hotspot and windows was able to login tailscale
i think due to some firewall restrictions in my office network the initial handshake is not working so not able to login.

**Comment by EselDompteur** [comment]
my success:

open system tray -> right click on tail scale -> disconnect -> exit
open system control -> find installed apps -> find tail scale -> remove application
open task manager -> move to start tab -> find tail scale -> right mouse click -> open path -> right mouse click -> deactivate
in opened path -> kill *.exe or kill path also

restart system ... tail scale is gone clean. Perhaps there are trash in the registry. 

best regards and sorry for my poor english

**Comment by gr84fun** [comment]
Hi i have a similar issue to this, tailscale was acting up causing local traffic to be blocked. I closed tailscale and tried to open it again and got an error message, i tried to stop service and didn't work. Long story short i disabled the network adapter and traffic started flowing again.  

My question is i cannot uninstall or update tailscale it either just sits there forever doing nothing during uninstall or upgrade. What's the process to get around this issue without using uninstaller?

**Comment by aaronsql2019** [comment]
yeah, I think that a simple install of tailscale just broke my f'in TPM. It doesn't show up. Not in the BIOS. I've uninstalled.

I wish you guys had tech support.

**Comment by awly** [comment]
We have uninstall instructions in https://tailscale.com/kb/1069/uninstall.

The local Tailscale Service on the system can be crashlooping due to a bug, we've definitely had releases with those slipping through. The latest version of Tailscale right now (1.92.5) does not crash to the best of my knowledge.
Unfortunately the reports here and comments are spread too far in time to pin any release or bug. I'm gonna close this issue, but feel free to reopen with any recent examples.

@aaronsql2019 I don't really know how Tailscale would manage to remove your TPM from the BIOS. If you have any more details, we do have tech support at https://tailscale.com/contact/support#support-form
